mlir/test/rocmlir-driver/populate_pv_with_gpu.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: rocmlir-gen --arch gfx908  -p -mfma=on -dot=on -atomic_add=on -pv_with_gpu | FileCheck %s
+// RUN: rocmlir-gen --arch gfx908  -p -mfma=on -dot=on -atomic_add=on -pv_with_gpu -disable-split-k-for-tuning | FileCheck %s
```

Comment:
Ah.

Note to future us: actually update these tests

Reply:
Good point. Done